[ERROR|kakaostoryCrawlerBot.py:159] 2019-01-25 16:36:45,841 > list index out of range
Traceback (most recent call last):
  File "C:/Users/tenspace/Desktop/crawlerBot_package_SCI/NotUsingJSONDATAType/kakaostoryCrawlerBot.py", line 136, in kakao_story_crawler_start
    'div.story_widgets > div[data-part-name=myInfo] > div > h3')[0].text
IndexError: list index out of range
[ERROR|kakaostoryCrawlerBot.py:159] 2019-01-25 16:38:29,647 > list index out of range
Traceback (most recent call last):
  File "C:/Users/tenspace/Desktop/crawlerBot_package_SCI/NotUsingJSONDATAType/kakaostoryCrawlerBot.py", line 136, in kakao_story_crawler_start
    'div.story_widgets > div[data-part-name=myInfo] > div > h3')[0].text
IndexError: list index out of range
[ERROR|kakaostoryCrawlerBot.py:457] 2019-01-25 16:38:55,692 > 'int' object has no attribute 'replace'
[ERROR|kakaostoryCrawlerBot.py:159] 2019-01-25 16:38:59,485 > list index out of range
Traceback (most recent call last):
  File "C:/Users/tenspace/Desktop/crawlerBot_package_SCI/NotUsingJSONDATAType/kakaostoryCrawlerBot.py", line 136, in kakao_story_crawler_start
    'div.story_widgets > div[data-part-name=myInfo] > div > h3')[0].text
IndexError: list index out of range
[ERROR|kakaostoryCrawlerBot.py:457] 2019-01-25 16:39:25,659 > 'int' object has no attribute 'replace'
[ERROR|kakaostoryCrawlerBot.py:159] 2019-01-25 16:39:29,428 > list index out of range
Traceback (most recent call last):
  File "C:/Users/tenspace/Desktop/crawlerBot_package_SCI/NotUsingJSONDATAType/kakaostoryCrawlerBot.py", line 136, in kakao_story_crawler_start
    'div.story_widgets > div[data-part-name=myInfo] > div > h3')[0].text
IndexError: list index out of range
[ERROR|kakaostoryCrawlerBot.py:159] 2019-01-25 17:37:42,415 > list index out of range
Traceback (most recent call last):
  File "C:/Users/tenspace/Desktop/crawlerBot_package_SCI/NotUsingJSONDATAType/kakaostoryCrawlerBot.py", line 136, in kakao_story_crawler_start
    'div.story_widgets > div[data-part-name=myInfo] > div > h3')[0].text
IndexError: list index out of range
[ERROR|kakaostoryCrawlerBot.py:458] 2019-01-25 17:38:22,189 > 'int' object has no attribute 'replace'
